apollo-router-core/src/plugin_utils/structures/router_request.rs

```diff
@@ -0,0 +1,66 @@
+use std::sync::Arc;
+
+use http::Request;
+use serde_json_bytes::{ByteString, Value};
+
+use crate::{Context, Object};
+
+#[derive(Default, Clone)]
+pub struct RouterRequestBuilder {
```

Comment:
I know we use `TypedBuilder` elsewhere. Could we use it here to reduce boilerplate?

Reply:
oh indeed, sorry I think you mentioned that before and i totally forgot to try to adress it, lemme check!

Reply:
fixed in d9fcb56